playbooks/10 - SP - Connector Health Monitoring/Notify Connector Health Status.json

```diff
@@ -540,5 +542,66 @@
     "deletedAt": null,
     "recordTags": [
         "Scheduled"
+    ],
+    "schedules": [
+        {
+            "id": "gAAAAABjEIf1X_WXBKODp1LEIuFPMN5vOXs6pUqoX0nvVYxgadugkHQttByZPrPtxSRjWLGVjdqamrzb5eUwXDIcHipRO9WkIA==",
+            "crontab": {
+                "id": 10,
+                "minute": "0",
+                "hour": "9",
+                "day_of_week": "*",
+                "day_of_month": "*",
+                "month_of_year": "*",
+                "timezone": "UTC"
+            },
+            "interval": null,
+            "name": "Notify Connector Health Status",
+            "task": "workflow.tasks.periodic_task",
+            "args": "[]",
+            "kwargs": {
+                "wf_iri": "\/api\/3\/workflows\/7ae88bb1-96b4-4728-9317-a810dec910d7",
+                "priority": {
+                    "@id": "\/api\/3\/picklists\/2b563c61-ae2c-41c0-a85a-c9709585e3f2",
+                    "@type": "Picklist",
+                    "itemValue": "Medium",
+                    "orderIndex": 1,
+                    "color": null,
+                    "icon": null,
+                    "listName": "\/api\/3\/picklist_names\/e104ef72-11b4-4d0c-be0e-e1cf3b87b5f2",
+                    "uuid": "2b563c61-ae2c-41c0-a85a-c9709585e3f2",
+                    "id": 111,
+                    "importedBy": []
+                },
+                "exit_if_running": true,
+                "createUser": "\/api\/3\/people\/3451141c-bac6-467c-8d72-85e0fab569ce",
+                "name": "Notify Connector Health Status",
+                "description": "Notify Connector Health Status",
+                "auth": {
+                    "auth_method": "CS HMAC"
+                },
+                "referenceid": null,
+                "schedule_entry_name": "Notify Connector Health Status",
+                "schedule_id": "661d5f81-2558-44ff-b301-c26f544cf660",
+                "start_time": null,
+                "expires": null
+            },
+            "queue": null,
+            "exchange": null,
+            "routing_key": null,
+            "headers": "{}",
+            "priority": 3,
+            "expires": null,
+            "expire_seconds": null,
+            "one_off": false,
+            "start_time": null,
+            "enabled": false,
```

Comment:
is this flag correct, should it be true? so when the customer installs the solution pack the schedule is activated or this is intentional.

Reply:
Intentional. We will document it..